src/main/java/com/voicepin/flow/client/VerifyStream.java

```diff
@@ -0,0 +1,68 @@
+package com.voicepin.flow.client;
+
+import com.voicepin.flow.client.calls.Call;
+import com.voicepin.flow.client.calls.VerifyResultCall;
+import com.voicepin.flow.client.calls.VerifyStreamCall;
+import com.voicepin.flow.client.data.SpeechStream;
+import com.voicepin.flow.client.exception.FlowClientException;
+import com.voicepin.flow.client.request.VerifyResultRequest;
+import com.voicepin.flow.client.request.VerifyStreamRequest;
+import com.voicepin.flow.client.result.VerifyInitResult;
+import com.voicepin.flow.client.result.VerifyResult;
+import com.voicepin.flow.client.result.VerifyStreamResult;
+
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Provides method for getting current result of verification process and setting listener
+ * on complete process.
+ *
+ * @author Lukasz Warzecha
```

Comment:
Provides current verification results (while still streaming) and accepts a final success/error callback.